sdlbook.c

```diff
@@ -144,6 +144,14 @@ static void init_gfx() {
 	if(!spritesheet_init(&ss_font, bmp_font, FONT_W, FONT_H)) dprintf(2, "oops\n");
 }
 
+static int get_page_bottom() {
+#ifndef OFF_SMOOTH
+	return page_dims.h;
+#else
+	return page_dims.h/2;
```

Comment:
page_dims.h / 2 never makes any sense - it was also the reason your previous PR only worked somewhat. page_dims is always the dimensions of curr_page, not both pages combined.